core/src/mindustry/world/blocks/power/PowerNode.java

```diff
@@ -112,6 +112,12 @@ public void setBars(){
             (UI.formatAmount((int)entity.power.graph.getLastPowerStored())), UI.formatAmount((int)entity.power.graph.getLastCapacity())),
             () -> Pal.powerBar,
             () -> Mathf.clamp(entity.power.graph.getLastPowerStored() / entity.power.graph.getLastCapacity())));
+
+        bars.add("connections", entity -> new Bar(() ->
+        Core.bundle.format("bar.powerlines", entity.power.links.size, maxNodes),
+            () -> Pal.ammo,
```

Comment:
@Anuken accept this suggestion to make it use a/the green bar, it might me more fitting given the context:
```suggestion
            () -> Pal.items,
```